flexmeasures/api/common/schemas/sensor_data.py

```diff
@@ -13,6 +15,42 @@
 from flexmeasures.data.schemas.times import AwareDateTimeField, DurationField
 
 
+def disambiguate_deserialization(data, _):
```

Comment:
No need to use their generic naming here, let's be clearer ― maybe:

`def select_schema_to_ensure_list_of_floats(values: Union[List[float], float]) -> Union[fields.List, SingleValueSchema]`